src/ui/scss/component/_comments.scss

```diff
@@ -45,3 +45,8 @@
   flex-basis: 200px;
   text-align: right;
 }
+
+.comment__char-count {
+  align-self: flex-end;
+  font-size: 12px;
```

Comment:
Can you change this to `font-size: var(--font-label);`

Reply:
Will do